apps/src/templates/projects/submitProjectDialog/submitProjectApi.ts

```diff
@@ -9,10 +9,29 @@ interface SubmissionStatusResponse {
 
 export type SubmissionStatusType = ValueOf<typeof ProjectSubmissionStatus>;
 /**
- * TODO: Sends a post request to submit the project.
+ * Sends a post request to submit the project to be considered for the featured project gallery.
  */
-export async function submitProject(submissionDescription: string) {
-  console.log('submitProject', submissionDescription);
+export async function submitProject(
```

Comment:
nit: might be good to explicitly type the return here (i.e. `Promise<SubmissionStatusType>`) since the HttpClient POST returns a `Response` object with an untyped `value`

Reply:
Oops sorry, I thought this was the API for fetching the submission status - in which case feel free to disregard!